docs/services/data/capture-sync.md

```diff
@@ -1,29 +1,37 @@
 ---
-title: "Configure Data Capture"
-linkTitle: "Data Capture"
-description: "Configure data capture to save data from components, services, or remote parts."
+title: "Data Capture and Sync"
+linkTitle: "Data Capture and Sync"
+description: "Configure the data manager to capture data from your components and services and sync it to the cloud."
 weight: 10
 type: "docs"
-tags: ["data management", "cloud", "sync"]
+tags: ["data management", "cloud", "sync", "capture"]
 icon: true
 images: ["/services/icons/data-capture.svg"]
 no_list: true
 aliases:
-  - "/services/data/capture/"
-  - "/data/capture/"
-  - "/build/micro-rdk/data_management/"
-# SME: Alexa Greenberg
+  - /services/data/capture/
+  - /data/capture/
+  - /build/micro-rdk/data_management/
+  - /services/data/capture/
+  - /services/data/cloud-sync/
+  - /data/cloud-sync/
 no_service: true
 ---
 
-The data management service captures data from one or more resources.
+The data management service captures data from one or more {{< glossary_tooltip term_id="resource" text="resources" >}} locally, and syncs it to cloud storage when a connection to the cloud is available.
+You can configure which data you want to capture, as well as the capture rate and the sync frequency.
+
+Data capture is frequently used with cloud sync.
+However, if you want to manage your machine's captured data yourself, you can enable only data capture without cloud sync.
 
 Get started with a quickstart guide or keep reading for more details.
 
 {{< cards >}}
 {{< card link="/get-started/collect-data/" class="green">}}
 {{< /cards >}}
 
+## Data capture details
+
 {{< tabs >}}
 {{% tab name="viam-server" %}}
 
```

Comment:
Why would this be on the viam-server tab specifically? Is this part of the assumption that all micro-viam-server users will use sync?

Reply:
I think you asked the data team about this right? If you capture data using viam-micro-server, you have captured it but you cannot access it. If you disconnect the device, it's gone. There's no way of getting at the data except to use cloud sync. So there's no point of using data capture without cloud sync

Reply:
I see what you're saying. However, I'm not sure about the disconnecting part?? If it's in flash memory, isn't still there?

Reply:
I am now confused because the internet says esp32 flash memory persists when power is cycled, but we have the line here in the docs to the contrary. Do you know for sure? Otherwise I'll ask about that.

Reply:
Hmm - I'm fairly certain it was SME-reviewed and that's what they said but what you say makes sense in that it should still be there. Maybe upon restarting `viam-micro` wipes flash storage or something.

I just re-read the convo you had with Gautham in the data management channel. He says "you won't have any means to retrieve it". By all means confirm it but if the answer is "technically you can do this but it's not useful in any way because you then cannot get the data or it's not usable because it's stored in a compressed format" we're not documenting it.

Reply:
I put these two sentences on the viam-server page anyway, so I think it's fine in its current state. I will double-check about whether the storage disappears when power cycling.